type.proto

```diff
@@ -75,3 +75,8 @@ message Object {
   Prediction prediction = 103;
   repeated int32 zone_ids = 104;
 }
+
+message ProfilingResult {
+  map<string, uint64> processing_times = 1;
+  map<string, uint64> mem_usages = 2;
```

Comment:
What are the strings? The algo node uid?

Reply:
The associated profiling block: e.g. "GroundDetector-Process"